### ISSUE TITLE
Add captions to brand store tables

### ISSUE THREAD

**Issue by steverydz** [canonical/snapcraft.io#4094]
For accessibility reasons tables should have captions. A caption is a description or a title explaining to the user what the purpose of the table is. In the case of the brand store this will not need to be visible to sighted users.
